lib/Controller/SAMLController.php

```diff
@@ -85,13 +85,34 @@ public function __construct($appName,
 	 */
 	private function autoprovisionIfPossible(array $auth) {
 		$uidMapping = $this->config->getAppValue('user_saml', 'general-uid_mapping');
+		$displaynamefirstpartMapping = $this->config->getAppValue('user_saml','general-displaynamefirstpart_mapping');
+		$displaynamesecondpartMapping = $this->config->getAppValue('user_saml','general-displaynamesecondpart_mapping');
```

Comment:
Yes. I just finished the automated integration test suite for Shibboleth. I'll see if I can find some time to dig into this here. :)